chef/cookbooks/nagios/templates/default/nrpe.cfg.erb

```diff
@@ -13,37 +13,37 @@ include_dir=/etc/nagios/nrpe.d/
 allowed_hosts=<%= @mon_host.join(',') %>
 <% end -%>
 
-command[check_users]=/usr/lib<%= @lib64 %>/nagios/plugins/check_users -w 20 -c 30
-command[check_load]=/usr/lib<%= @lib64 %>/nagios/plugins/check_load -w <%= node[:nagios][:checks][:load][:warning] %> -c <%= node[:nagios][:checks][:load][:critical] %>
-command[check_all_disks]=/usr/lib<%= @lib64 %>/nagios/plugins/check_disk -w 8% -c 5% -A -x /dev/shm -X nfs -i /boot
-command[check_zombie_procs]=/usr/lib<%= @lib64 %>/nagios/plugins/check_procs -w 5 -c 10 -s Z
-command[check_total_procs]=/usr/lib<%= @lib64 %>/nagios/plugins/check_procs -w 500 -c 800
-command[check_swap]=/usr/lib<%= @lib64 %>/nagios/plugins/check_swap -w '50%' -c '25%'
-command[check_mem]=/usr/lib<%= @lib64 %>/nagios/plugins/check_mem.sh -w <%= node[:nagios][:checks][:memory][:warning] %> -c <%= node[:nagios][:checks][:memory][:critical] %> -p
-command[check_chef_client]=/usr/lib<%= @lib64 %>/nagios/plugins/check_procs -w 1:2 -c 1:2 -C ruby -a chef-client
-command[check_smtp]=/usr/lib<%= @lib64 %>/nagios/plugins/check_smtp -H <%= node[:nagios][:checks][:smtp_host] %>
-command[check_nginx]=/usr/lib<%= @lib64 %>/nagios/plugins/check_procs -w 2:3 -c 1:5 -C nginx
-command[check_sphinx]=/usr/lib<%= @lib64 %>/nagios/plugins/check_procs -c 1:1 -C searchd
+command[check_users]=/usr/lib/nagios/plugins/check_users -w 20 -c 30
+command[check_load]=/usr/lib/nagios/plugins/check_load -w <%= node[:nagios][:checks][:load][:warning] %> -c <%= node[:nagios][:checks][:load][:critical] %>
+command[check_all_disks]=/usr/lib/nagios/plugins/check_disk -w 8% -c 5% -A -x /dev/shm -X nfs -i /boot
+command[check_zombie_procs]=/usr/lib/nagios/plugins/check_procs -w 5 -c 10 -s Z
+command[check_total_procs]=/usr/lib/nagios/plugins/check_procs -w 500 -c 800
+command[check_swap]=/usr/lib/nagios/plugins/check_swap -w '50%' -c '25%'
+command[check_mem]=/usr/lib/nagios/plugins/check_mem.sh -w <%= node[:nagios][:checks][:memory][:warning] %> -c <%= node[:nagios][:checks][:memory][:critical] %> -p
+command[check_chef_client]=/usr/lib/nagios/plugins/check_procs -w 1:2 -c 1:2 -C chef-client
+command[check_smtp]=/usr/lib/nagios/plugins/check_smtp -H <%= node[:nagios][:checks][:smtp_host] %>
+command[check_nginx]=/usr/lib/nagios/plugins/check_procs -w 2:3 -c 1:5 -C nginx
+command[check_sphinx]=/usr/lib/nagios/plugins/check_procs -c 1:1 -C searchd
```

Comment:
you could simply s,/usr/lib<%= @lib64 %>/nagios/plugins/,<%= @plugin_dir %>/,

I think